### ISSUE TITLE
Font awesome icon problem

### ISSUE THREAD

**Issue by erichard** [Viima/jquery-comments#30]
Hello, 
Icons from fontawesome are broken when I integrate jquery-comments. 
![icon_problem](https://cloud.githubusercontent.com/assets/651041/12348792/b5f41902-bb6a-11e5-8bc4-a66db7dee91f.png)

The problem come from the `font-family: inherit;` in the firsts lines of the css. If I remove this line everything work fine.

**Comment by jjtykkyl** [comment]
Did you load the font-awsesome.css after the jquery-comments.css as it is supposed to?

**Comment by erichard** [comment]
I made a workaround by forcing the font family on:
```
.jquery-comments i.fa {
  font: normal normal normal 14px/1 FontAwesome;
}
```

My font-awesome library is included as part of an admin theme that I cannot change.

**Comment by jjtykkyl** [comment]
I'm happy to let you know that this bug has been fixed in version 1.1.0
